### ISSUE TITLE
Wrong conversion of list initializacion

### ISSUE THREAD

**Issue by angelolloqui** [angelolloqui/SwiftKotlin#68]
This Swift:
```
var temporaryPlayers = [String]()
```

Currently translates to:

```
var temporaryPlayers = listOf(String)()
```

But should be:
```
var temporaryPlayers = listOf<String>()
```

**Comment by yohanhyunsungyi** [comment]
Hello,

I have fixed this bug and created a pull request.
Please review it.
https://github.com/angelolloqui/SwiftKotlin/pull/95

Thank you

**Comment by angelolloqui** [comment]
Fixed with #96
